app/views/components/_published-dates.html.erb

```diff
@@ -0,0 +1,30 @@
+<%
+  published ||= false
+  history ||= false
+  last_updated ||= false
+  link_to_history ||= false
+%>
+
+<div class="app-c-published-dates" <% if history%>id="history" data-module="toggle"<% end %>>
+  <% if published %>
+    Published <%= published %>
+  <% end %>
+  <% if last_updated %>
+    <br />Last updated <%= last_updated %>
+    <% if link_to_history && !history %>
+      &mdash; <a href="#history">see all updates</a>
+    <% elsif history %>
+      <a href="#full-history" data-controls="full-history" data-expanded="false">&#43; full page history</a>
```

Comment:
I noticed on mobile that the `+` wrapped to a new line. Might be good to include a `nbsp;` in `&#43; full`